INSTALL.md

```diff
@@ -127,11 +127,12 @@ On SUSE:
 
 ## Windows details
 
-Open a [developer command prompt](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
-or in the command line (`cmd.exe`) call:
+Open a command prompt window (`cmd.exe`) and call:
 
     call "C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\Tools\VsDevCmd.bat" -arch=x64
 
+The command above is required to while using the 64-bit version of skia. When compiling with the 32-bit version, it is possible to open a [developer command prompt](https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs) instead.
```

Comment:
Extraneous "to" in the first sentence.

Reply:
Thank you, I did not notice that! I will fix it soon!